src/backend/executor/nodeDML.c

```diff
@@ -261,6 +286,7 @@ ExecInitDML(DML *node, EState *estate, int eflags)
 	dmlstate->junkfilter = ExecInitJunkFilter(node->plan.targetlist,
 			dmlstate->ps.state->es_result_relation_info->ri_RelationDesc->rd_att->tdhasoid,
 			dmlstate->cleanedUpSlot);
+	estate->es_result_relation_info->ri_junkFilter = dmlstate->junkfilter;
```

Comment:
Why we need this? May be add some words about it into commit/PR description?

Reply:
it was needed for EPQ logic, but since it was decided to abandon it in this PR, it is no longer needed.

Reply:
I suggest to move it to next EPQ patch